chart/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+---
+annotations:
+  charts.openshift.io/name: Red Hat Developer Hub Setup
```

Comment:
I remember RHDH also has operator. so we might want to discuss to use operator or helm chart for the installation

Reply:
RHDH is going to be added to this repo (most likely as a chart template like rhtap [here](https://github.com/redhat-appstudio/rhtap-installer/tree/main/chart/templates/developer-hub)) as part of https://issues.redhat.com/browse/DEVHAS-753